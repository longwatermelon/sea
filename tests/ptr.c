fn f(addr: int) -> int {
    return *addr;
}

fn main() -> int {
    let x:int=5;
    let y:int=&x;
    return 1+*f(&y)+1;
}
