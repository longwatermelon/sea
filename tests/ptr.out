RETURN_CODE:7
STDOUT:
